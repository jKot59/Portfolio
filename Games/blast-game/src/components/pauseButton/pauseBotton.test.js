import React from 'react';
import PauseButton from './pauseButton';
import renderer from 'react-test-renderer';

// создаем тест сьют
describe('Testing pause button', () => {
    // создаем тест кейс
    it('Pause button have rendered correctly', () => {
        // создаем копию компонента
        const pauseButton = renderer.create(<PauseButton/>).toJSON()
        // создаем снимок компонента
        expect(pauseButton).toMatchSnapshot()
    })
})