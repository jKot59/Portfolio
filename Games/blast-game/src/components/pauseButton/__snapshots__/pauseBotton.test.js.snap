// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing pause botton Pause botton have rendered correctly 1`] = `
<div
  className="sc-bdnxRM dLwCkD"
/>
`;

exports[`Testing pause botton Pause button have rendered correctly 1`] = `
<div
  className="sc-bdnxRM dLwCkD"
/>
`;

exports[`Testing pause button Pause button have rendered correctly 1`] = `
<div
  className="sc-bdnxRM dLwCkD"
/>
`;
